eduroam/nmcli.md

```diff
@@ -28,22 +28,22 @@ All commands starting with `$` can be used as your standard user. All commands u
 
 2. Grab the name of the wireless network interface  that you want to use for this connection. You will need this throughout this setup process. You can list all network interfaces by running the command:
 ```
-$ nmcli
+$ nmcli device show
 ```
+>Check the one whose `GENERAL.TYPE` is `wifi`. Most commonly, this is `wlan0` or `wlp5s0`.
 3. Open a terminal and use nmcli to create a new wifi connection on your wireless network interface that connects to the `eduroam` network:
 ```
-$ nmcli connection add type wifi ifname <your interface> con-name <connection name> ssid eduroam
+$ nmcli connection add type wifi ifname <your interface> con-name eduroamWiFi ssid eduroam
 ```
-4. then navigate to the NetworkManager/`nmcli` configuration directory:
+4. Open the `eduroamWiFi.nmconnection` file in the editor of your choice *as root*. If you do not open it as root, the changes will not be saved.
 ```
-# cd /etc/NetworkManager/system-connections/
+# vim /etc/NetworkManager/system-connections/eduroamWiFi.nmconnection
 ```
-5. Open the `eduroam.nmconnection` file in the editor of your choice *as root*. If you do not open it as root, the changes will not be saved.
-
-6. Edit the file so that it contains the following lines. The file will contain most of these lines already. The `uuid` field will already be filled in, do not edit this field.
+>Substitute `vim` for editor of choice
```

Comment:
i think this was the reason it was split into two instructions, so that there wasnt a need to assume what editor to use for the command. IMO its clearer as two steps. people who know what they are doing can just skim these steps and be like "oh yeah i can just open this in one step"

Reply:
Perhaps. This is a situation where I could see the logic going either way to be honest. Let me explain my case though:

You would have to actually switch to root user to cd into that directory (/etc/NetworkManager/system-connections). To avoid writing an unnecessary extra step where you use `sudo su` to switch users, I think it's just easier to use `sudo $EDITOR /path/to/file`, which would also eliminate a step.

If you'd like, you can simply tell the user to switch to root user here but I think it's just clearer to do it in one command.

Reply:
is $EDITOR something that actually exists? or just a placeholder like <your-interface>?

Reply:
`$EDITOR` has always been defined on my systems. I just said that to explain my point though.
```bash
sudo vim /etc/NetworkManager/system-connections/eduroamWiFi.nmconnection
```
>Substitute `vim` for editor of choice

is more succinct than:
```bash
sudo su
cd /etc/NetworkManager/system-connections
sudo <your-editor> <con-name>.nmconnection
```

This topic is a bit pedantic though. This is not my project so, of course, I do not have the final say for what passes/doesn't pass. However, this is the guide **I wish I** would have read. This is simply my two cents for what should be here. Feel free to make your edits/reversions to what you think suits more people the best.

Reply:
other than maybe changing vim to something more beginner friendly (nano), i think your solution is better than what is here already

Reply:
That makes sense. I just pushed a change altering the placeholder editor to `nano`.